src/models/testSuite.ts

```diff
@@ -1,41 +1,35 @@
 import {EntityMap} from '@models/entityMap';
-import {ObjectRef} from '@models/objectRef';
 import {TestSuiteExecution} from '@models/testSuiteExecution';
 
 import {Execution} from './execution';
 import {Variables} from './variable';
 
-export type TestSuiteStepDelay = {
-  duration: number;
-};
-
-export type TestSuiteStepExecuteTest = {
-  namespace?: string;
-  name: string;
-  type: string;
-};
-
-export interface TestSuiteStepAbstract {
-  stopTestOnFailure: boolean;
-  // Used for form
-  id?: string | number;
+export interface AdditionalStepFields extends TestSuiteExecuteStep {
+  stopTestOnFailure?: boolean;
+  id?: string;
 }
 
-export interface TestSuiteStepWithExecute extends TestSuiteStepAbstract {
-  execute: TestSuiteStepExecuteTest;
+export interface TestSuiteExecuteStep {
+  delay?: string;
+  test?: string;
+}
+export interface TestSuiteStep {
+  stopTestOnFailure: boolean;
+  execute: TestSuiteExecuteStep[];
 }
 
-export interface TestSuiteStepWithDelay extends TestSuiteStepAbstract {
-  delay: TestSuiteStepDelay;
+export interface LocalStep extends AdditionalStepFields {
```

Comment:
Could we keep types naming from OpenAPI spec? Will be later simpler to understand it and search for it